index.html

```diff
@@ -63,28 +63,37 @@ <h2 id="project_tagline"></h2>
 <pre>$ java -jar publish.war -h
 You passed help flag.
 usage: java -jar
- -en,--exchange_name &lt;arg&gt;    exchange name, default is amq.direct
- -f,--content_file &lt;arg&gt;      event content file
- -h,--help                    show help.
- -json,--json_content &lt;arg&gt;   event content in json string. The value can
-                              also be a dash(-) and the json will be read
-                              from the output of other programs if piped.
- -mb,--message_bus &lt;arg&gt;      host of message bus to use, default is
-                              127.0.0.1
- -np,--non_persistent         remove persistence from message sending
- -port,--port &lt;arg&gt;           port to connect to message bus
- -rk,--routing_key &lt;arg&gt;      routing key, mandatory</pre>
+ -en,--exchange_name &lt;arg&gt;      exchange name, default is amq.direct
+ -f,--content_file &lt;arg&gt;        event content file
+ -h,--help                      show help.
+ -json,--json_content &lt;arg&gt;     event content in json string. The value can
+                                also be a dash(-) and the json will be read
+                                from the output of other programs if piped. Multiple
+                                json events are also supported.
+ -mb,--message_bus &lt;arg&gt;        host of message bus to use, default is
+                                127.0.0.1
+ -np,--non_persistent           remove persistence from message sending
+ -port,--port &lt;arg&gt;             port to connect to message bus
+ -ud,--user_domain_suffix &lt;arg&gt; user domain suffix
+ -d,--debug &lt;arg&gt;               enable debug traces
+ -mp,--message_protocol &lt;arg&gt;   name of messaging protocol to be used, e.g. eiffel3,
+                                eiffelsemantics, default is eiffelsemantics</pre>
```

Comment:
We are not passing any -domain in CLI. The domain information is procured from application.yml file.